compiler/rustc_hir_typeck/src/writeback.rs

```diff
@@ -364,12 +363,18 @@ impl<'cx, 'tcx> Visitor<'tcx> for WritebackCx<'cx, 'tcx> {
         }
     }
 
-    fn visit_infer(&mut self, inf: &'tcx hir::InferArg) {
-        intravisit::walk_inf(self, inf);
+    fn visit_infer(
+        &mut self,
+        inf_id: HirId,
+        inf_span: Span,
+        _kind: InferKind<'cx>,
+    ) -> Self::Result {
+        self.visit_id(inf_id);
+
         // Ignore cases where the inference is a const.
```

Comment:
Can u tweak this comment to say that we don't currently *need* to write consts to the typeck results.

Saying "ignore" here makes it seem like we're skipping something important 🤔